rosbag2_py/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rosbag2_py</name>
+  <version>0.2.4</version>
+  <description>Python API for rosbag2</description>
+  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
```

Comment:
Can you add `<maintainer email="ros-tooling@googlegroups.com">ROS Tooling Working Group</maintainer>`